davis_ros_driver/src/driver.cpp

```diff
@@ -677,39 +679,31 @@ void DavisRosDriver::readout()
                         // Get full timestamp and addresses of first event.
                         caerPolarityEvent event = caerPolarityEventPacketGetEvent(polarity, j);
 
-                        dvs_msgs::Event e;
-                        e.x = caerPolarityEventGetX(event);
-                        e.y = caerPolarityEventGetY(event);
-                        e.ts = reset_time_
-                                + ros::Duration().fromNSec(caerPolarityEventGetTimestamp64(event, polarity) * 1000);
-                        e.polarity = caerPolarityEventGetPolarity(event);
+                        int e_x = caerPolarityEventGetX(event);
+                        int e_y = caerPolarityEventGetY(event);
+                        bool e_polarity = caerPolarityEventGetPolarity(event);
+                        int pos = e_y * event_frame_msg.step + e_x * 3 + e_polarity;
 
-                        if(j == 0)
-                        {
-                            event_array_msg->header.stamp = e.ts;
-                        }
+                        if (event_frame_msg.data[pos] < 255) // prevent overflow
```

Comment:
Should there be an else statement with some warning in case of Overflow happening.